OurUmbraco.Site/web.vsts.config

```diff
@@ -579,6 +579,26 @@
                     <match url="rss/wiki*" />
                     <action type="Redirect" url="/Documentation/" redirectType="Permanent" appendQueryString="true" />
                 </rule>
+		<rule name="Setup" patternSyntax="Wildcard">
+            	    <match url="documentation/Getting-Started/Setup/*" />
+                    <action type="Redirect" url="/Documentation/Fundamentals/Setup/" redirectType="Permanent" appendQueryString="true" />
+                </rule>
+		<rule name="Backoffice" patternSyntax="Wildcard">
+            	    <match url="documentation/Getting-Started/Backoffice/*" />
+                    <action type="Redirect" url="/Documentation/Fundamentals/Backoffice/" redirectType="Permanent" appendQueryString="true" />
+                </rule>
+		<rule name="Data" patternSyntax="Wildcard">
+            	    <match url="documentation/Getting-Started/Data/*" />
+                    <action type="Redirect" url="/Documentation/Fundamentals/Data/" redirectType="Permanent" appendQueryString="true" />
+                </rule>
+		<rule name="Design" patternSyntax="Wildcard">
+            	    <match url="documentation/Getting-Started/Design/*" />
+                    <action type="Redirect" url="/Documentation/Fundamentals/Design/" redirectType="Permanent" appendQueryString="true" />
+                </rule>
+		<rule name="Design" patternSyntax="Wildcard">
```

Comment:
This needs to be changed to:
`<rule name="Code" patternSyntax="Wildcard">`

After that is fixed, everything looks perfect ✨